src/components/LoadingMessage/LoadingMessage.less

```diff
@@ -0,0 +1,33 @@
+.lucid-LoadingMessage {
+
+	.lucid-Icon {
+		display: inline-block;
+	}
+
+	@LoadingMessage-padding: @size-standard;
+
+	line-height: 1em;
+	background-color: @color-white;
+	position: relative;
+	display: inline-block;
+	text-align: center;
+	border: @border-lightBorder;
+	border-radius: @size-borderRadius;
+	padding: @LoadingMessage-padding;
+	width: 135px + (@LoadingMessage-padding * 2);
```

Comment:
if you use `box-sizing: border-box`, you don't need to add the side paddings to the width here